frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,396 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { pushState } from 'redux-router';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import Notification from '../containers/Notification';
+
+import Checkbox from '../components/Checkbox';
+import CustomTextArea from '../components/CustomTextArea';
+import ImagePicker from '../components/ImagePicker';
+import Input from '../components/CustomTextArea';
+import PublicFooter from '../components/PublicFooter';
+import Select from '../components/Select';
+
+import appendGroupForm from '../actions/form/append_group';
+import createGroup from '../actions/groups/create';
+import notify from '../actions/notification/notify';
+import uploadImage from '../actions/images/upload';
+import validateSchema from '../actions/form/validate_schema';
+
+import groupSchema from '../joi_schemas/group';
+
+import i18n from '../lib/i18n';
+
+const memberRole = 'MEMBER';
+
+export class Addgroup extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      appendGroupForm,
+      groupForm,
+      i18n
+    } = this.props;
+
+    const {
+      name,
+      slug,
+      currency,
+      website,
+      video,
+      image,
+      logo,
+      mission,
+      description,
+      longDescription,
+      whyJoin,
+      tags,
+      users1,
+      users2,
+      users3,
+      users4,
+      users5
```

Comment:
How about an even more hackier hack?

Store using `JSON.stringify()` for the users and a comma separated list for the array:

To Deflate
`[{...},{...},{...}].map( x => JSON.stringify(x) ).toString()`

To Inflate
`"{...},{...},{...}".split().map( x=> JSON.parse(x) )`